stereo/src/stereo_grabber.cpp

```diff
@@ -78,11 +78,11 @@ pcl::StereoGrabberBase::StereoGrabberImpl::StereoGrabberImpl(
 , frames_per_second_(frames_per_second)
 , repeat_(repeat)
 , running_(false)
+, pair_files_({pair_files})
 , time_trigger_(1.0 / static_cast<double>(std::max(frames_per_second, 0.001f)),
                 [this] { trigger(); })
 , valid_(false)
 {
-  pair_files_.push_back(pair_files);
   pair_iterator_ = pair_files_.begin();
 }
 
```

Comment:
You missed what cppcheck was actually complaining about: `pair_files_` in line 103

Reply:
Done